content/doc/book/installing/index.adoc

```diff
@@ -80,16 +80,18 @@ link:https://docs.docker.com/get-started/[Getting Started, Part 1: Orientation
 and setup] page.
 
 Docker's fundamental platform and container design means that a single Docker
-image (for any given application like Jenkins) can be run on any operating
-system (e.g. macOS, Linux and Windows) which is also running Docker.
+image (for any given application like Jenkins) can be run on any supported
+operating system (e.g. macOS, Linux and Windows) or cloud service (e.g. AWS and
+Azure) which is also running Docker.
 
 
 ==== Installing Docker
 
 To install Docker on your operating system, visit the
-link:https://www.docker.com/[Docker] website and navigate to *Get Docker* at the
-top of their site to begin accessing the Docker download that is suitable for
-your operating system. Follow the installation instructions on their website.
+link:https://store.docker.com/search?type=edition&offering=community[Docker
+store] website and click the *Docker Community Edition* which is suitable for
+your operating system or cloud service. Follow the installation instructions on
+their website.
```

Comment:
Is it our place to recommend the community edition? We don't really care if it's Docker-EE or Docker-CE as long as Docker is installed?

Reply:
The link I sent takes people to the download page for Docker-CE, which supports macOS and Windows desktop versions that the Docker-EE edition doesn't.

Again, I'm catering for the 'new to the field' group, to whom I'm providing the 'easy path' to get Docker up and running as smoothly as possible.

I'll mention that Docker-EE can also be used, but that's as far as I'll describe things.